From d21247010c8e885916908e641481d1d915d8a690 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date: Mon, 13 May 2019 16:47:54 +0100
Subject: [PATCH 630/725] staging: vcsm-cma: Alter dev node permissions to 0666

Until the udev rules are updated, open up access to this node by
default.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>
---
 drivers/staging/vc04_services/vc-sm-cma/vc_sm.c | 2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/staging/vc04_services/vc-sm-cma/vc_sm.c
+++ b/drivers/staging/vc04_services/vc-sm-cma/vc_sm.c
@@ -1572,6 +1572,8 @@ static void vc_sm_connected_init(void)
 	sm_state->misc_dev.name = DEVICE_NAME;
 	sm_state->misc_dev.fops = &vc_sm_ops;
 	sm_state->misc_dev.parent = NULL;
+	/* Temporarily set as 666 until udev rules have been sorted */
+	sm_state->misc_dev.mode = 0666;
 	ret = misc_register(&sm_state->misc_dev);
 	if (ret) {
 		pr_err("vcsm-cma: failed to register misc device.\n");
